modules/sandboxed-containers-preparing-openshift-cluster.adoc

```diff
@@ -3,5 +3,102 @@
 // * sandboxed_containers/deploying_sandboxed_containers.adoc
 
 [id="sandboxed-containers-preparing-openshift-cluster_{context}"]
+= Preparing your cluster for {sandboxed-containers-first}
 
-= Preparing your cluster for OpenShift sandboxed containers
+Before you install {sandboxed-containers-first}, ensure that your {product-title} cluster meets the following requirements:
+
+* Your cluster must be installed on bare metal infrastructure with {op-system-first} workers.
+// To Do: Add link to bare metal instructions: ../../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[bare metal]
++
+[IMPORTANT]
+====
+* {sandboxed-containers-first} only supports {op-system} worker nodes. RHEL 7 or RHEL  8 nodes are not supported.
+* Nested virtualization is not supported.
+====
+* Your cluster must use installer-provisioned infrastructure.
+// To Do: Add link to installer provisioned infrastructure: ../../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install-overview[Installer provisioned infrastructure]
+
+[id="sandboxed-containers-additional-resource-requirements_{context}"]
+== Additional resource requirements for {sandboxed-containers-first}
+
+{sandboxed-containers-first} is a product that brings the ability to run workloads inside a sandboxed runtime, such as Kata Containers, to your {product-title} clusters. Each pod is represented by a virtual machine (VM). Each VM hosts a `kata-agent` process that acts as a supervisor for managing container workloads and processes running in those container. There are two additional processes that add more overhead:
+
+* `containerd-shim-kata-v2` is used to communicate with the pod.
+* `virtiofsd` handles host file system access on behalf of the guest.
+* Each VM runs in a `qemu` process and hosts a `kata-agent` process. 
+
+Each VM is configured with a default amount of memory. Additional memory is hot-plugged into the VM for containers that explicitly request memory.
+
+* If a container runs without a given memory resource, it is able to consume free memory. It will do so until the total memory used by the VM reaches the default allocation. The guest and its I/O buffers also consume memory.
+* If a container is given a specific amount of memory, then that memory is hot-plugged into the VM before the container starts.
+* If a memory limit is specified, then the workload is terminated if it consumes more memory than the limit. If no memory limit is specified, the kernel  that is running on the virtual machine might run out of memory. If the kernel runs out of memory it might terminate other process on the virtual machine.
+
+.Indicative memory sizes
+
+The table below lists some typical values for resource allocation.
+
+[cols="2,2"]
+|===
+|Resource |Value
+
+|Memory allocated by default to a virtual machine | 2G
```

Comment:
@sagidlow I believe that we decided to use 2Gi, 30Mi, etc for consistency with the rest of the documentation.